contributing/documentation/introduction_guide.rst

```diff
@@ -32,8 +32,8 @@ If you need to learn about a specific markup, head over to :doc:`our cheat sheet
 documentation of Odoo.
 
 .. important::
-   We kindly ask you to observe a set of :doc:`guidelines <rst_guidelines>` as you write :abbr:`RST
-   (reStructuredText)`. This ensures that you stay consistent with the rest of the documentation and
+   We kindly ask you to observe a set of :doc:`content <content_guidelines>` and :doc:`RST <rst_guidelines>` as you write documentation.
```

Comment:
```suggestion
   We kindly ask you to observe a set of :doc:`content <content_guidelines>` and :doc:`RST <rst_guidelines>` guidelines as you write documentation.
```
Woops

Reply:
I already added it right now but inside the RST link

Reply:
Outside would be better I think. You have a link on both so neither should include 'guidelines'.